src/inmanta/moduletool.py

```diff
@@ -912,12 +921,32 @@ def build(self, output_directory: str) -> str:
             # Copy module to temporary directory to perform the build
             build_path = os.path.join(tmpdir, "module")
             shutil.copytree(self._module.path, build_path)
+            if dev_build:
+                self._add_dev_build_tag_to_setup_cfg(build_path)
```

Comment:
If this is a build on a v2 module, doesn't this persist the change? Is there no `-Db my_tag` alternative for `setup.cfg` based builds?

Reply:
No. The module is first copied into a temporary build directory. Only the `setup.cfg` from this temporary directory is adjusted. The original one is left untouched.